sssom/parsers.py

```diff
@@ -53,10 +59,15 @@ def read_sssom_table(
     file_path: Union[str, Path],
     prefix_map: Optional[PrefixMap] = None,
     meta: Optional[MetadataType] = None,
+    mapping_predicates: Optional[List[str]] = None,
```

Comment:
i basically thought to not have the `mapping_predicates` option for all the `read_` methods - only for the `parse_` methods in `parsers.py`...